DocSum/docker_image_build/build.yaml

```diff
@@ -11,18 +11,57 @@ services:
       context: ../
```

Comment:
In the docker build yaml, keep the service name the same as docker image name.   
Please also add the new ui docker image build here.